docs/sdks/sdk-ref-ios.md

```diff
@@ -0,0 +1,198 @@
+
+## Functionality
+
+This SDK simplifies integration with EUID for any publishers who want to support EUID for apps running on iOS devices. The following table shows the functions it supports.
+
+| Encrypt Raw EUID to EUID Token | Decrypt EUID Token to Raw EUID | Generate EUID Token from DII | Refresh EUID Token | Map Personal Data to Raw EUID |
+| :--- | :--- | :--- | :--- | :--- |
+| &#8212; | &#8212; | &#9989; | &#9989; | &#8212; |
+
+The SDK for iOS is designed to generate and/or manage EUID identity on behalf of iOS apps. It enables EUID identity to be persisted across app lifecycles by securely storing the identity on a device via platform-native encryption tools.
+
+By default, the SDK automatically refreshes EUID identity based on expiration dates. However, you can disable this to allow implementing apps to manage the EUID identity lifecycle manually.
+
+## API Permissions
+
+To use this SDK, you'll need to complete the EUID account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page.
+You'll be granted permission to use specific functions offered by the SDK, and given credentials for that access.
+
+## SDK Version
+
+<!-- As of 7 May 2024 -->
+
+This documentation is for the SDK for iOS version 1.7.0 and later.
+
+For current and past release notes information, see [https://github.com/IABTechLab/uid2-ios-sdk/releases](https://github.com/IABTechLab/uid2-ios-sdk/releases).
+
+## GitHub Open-Source Repository
+
+This SDK is in the following open-source GitHub repository:
+
+- [https://github.com/IABTechLab/uid2-ios-sdk](https://github.com/IABTechLab/uid2-ios-sdk)
+
+## Minimum Requirements
+
+Minimum requirements for this SDK are as follows:
+
+- Xcode version: 15.0+
+- iOS	minimum target version: 13.0+
+- tvOS minimum target version: 13.0+
+- Swift version: 5.0+
+
+<!-- See also: [Requirements](https://github.com/IABTechLab/uid2-ios-sdk/blob/main/README.md#requirements). -->
+
+## Installation
+
+Install the iOS SDK via Swift Package Manager (SPM) or CocoaPods. There are three installation options:
+
+-   [Package.swift](#installing-with-packageswift)
+-   [Xcode](#installing-with-xcode)
+-   [CocoaPods](#installing-with-cocoapods)
+
+### Installing with Package.swift
+
+Add the following dependency to Package.swift:
+
+```js
+dependencies: [
+  .package(url: "https://github.com/IABTechLab/uid2-ios-sdk.git", from: "1.7.0"),
+]
+```
+
+### Installing with Xcode
+
+In the XCode user interface, under Package Dependencies, add the following entry for your apps:
+
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-sdk | `git@github.com:IABTechLab/uid2-ios-sdk.git` | Up to next major version: 1.7.0 < 2.0.0 |
+
+### Installing with CocoaPods
+
+Add the following entry in your `Podfile`:
+
+```
+pod 'UID2', '~> 1.2'
```

Comment:
Probably it should be `pod 'UID2', '~> 1.7'`

Reply:
Fixed. TYSM.